charts/admission-controller/tests/golden_template_test.yaml

```diff
@@ -0,0 +1,44 @@
+suite: Testing pre-generated values and best-known edge cases 
+templates:
+  - scanner/deployment.yaml
+  - scanner/configmap.yaml
+  - scanner/secret.yaml
+  - webhook/deployment.yaml
+  - webhook/configmap.yaml
+  - webhook/secret.yaml
+  - webhook/admissionregistration.yaml
+tests:
+  - it: check value of secure_API_TOKEN without local chart override
+    documentIndex: 0
+    set:
+      sysdig:
+        secureAPIToken: standard_token
+      global:
+        sysdig:
+          region: us2
+      clusterName: test-k8s
+      scanner:
+        enabled: true
+      webhook:
+        enabled: true
```

Comment:
i don't see `webhook.enabled `  key in the values at all

Reply:
Removed it. I must have assumed there was while running it good to know.